python/openchange/migration/directory.py

```diff
@@ -26,6 +26,24 @@
 import sys
 
 
+def _get_local_samdb(**kwargs):
+        from openchange.provision import get_local_samdb
+
+        lp = kwargs['lp']
+        creds = kwargs['creds']
+        names = kwargs['names']
+        return get_local_samdb(names, lp, creds)
+
+
+def _get_all_enabled_users(db, **kwargs):
```

Comment:
I wanted to leave them outside as utility functions for future provisions, if you two dont like the idea I will inline them .